source/content/modules-plugins-known-issues.md

```diff
@@ -173,6 +173,18 @@ You can modify this patch according to your needs, such as performing an operati
 **Issue**: This module requires modification of the site's `.htaccess` or `nginx.conf` file, which cannot be modified on the platform. While using `settings.php` can sometimes be effective as a means of implementing redirects, because `POST` data needs to be preserved, it is not possible to implement redirects at the application layer in a way that would allow this module to function as intended.
 <hr />
 
+### [Layout Builder](https://www.drupal.org/docs/8/core/modules/layout-builder)
+**Issue**: This core module returns an error when trying activate:
+
+```php
+Fatal error: require(): Failed opening required '/srv/bindings/bindingIDxxxxxxxxx/code/vendor/composer/../../web/core/lib/Drupal/Core/Layout/Icon/SvgIconBuilder.php'
+```
+
+This is due to the .gitignore file that excludes `Icon/SvgIconBuilder.php`.
+
+**Solution**: In the `.gitignore` file in your local install or build server, remove the `Icon` line so that file can be included in your push or deploys.
```

Comment:
Can we replace `your local install or build server` with the relative path to the `.gitignore` file in the code base?

Reply:
Not really sure, whatever would be easiest to understand in a user's perspective as long as they can easily locate the .gitignore file and make sure the missing gets included in their repo and pushed to their codebase

Reply:
@carl-alberto what would that path be?

Reply:
@alexfornuto It should be in the root path of the Drupal installation (/code folder). But it seems the ones affected here are versions prior to 8.7.5  https://github.com/pantheon-systems/drops-8/commit/6af7bf3045b168a6fa4b866189bf143aa12f85af and latest D7 https://github.com/pantheon-systems/drops-7/commit/2eb2fd38ca57178fb138326dda4abb6329c815bc. Another alternative solution is to this is upgrade to the latest Drupal version

Reply:
@carl-alberto I'm asking for you to provide the path from the project root to the file in question to be removed.

Reply:
@alexfornuto If I'm not mistaken

/code/vendor/composer/../../web/core/lib/Drupal/Core/Layout/Icon/SvgIconBuilder.php